sda-commons-server-weld-example/src/test/java/org/sdase/commons/server/weld/WeldExampleApplicationITest.java

```diff
@@ -1,22 +1,52 @@
 package org.sdase.commons.server.weld;
 
 import static io.dropwizard.testing.ConfigOverride.randomPorts;
+import static org.assertj.core.api.Assertions.assertThat;
 
+import io.dropwizard.Application;
 import io.dropwizard.Configuration;
-import org.assertj.core.api.Assertions;
+import io.dropwizard.testing.DropwizardTestSupport;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import javax.enterprise.inject.spi.CDI;
+import javax.ws.rs.core.MediaType;
+import org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider;
+import org.jboss.weld.junit5.auto.AddExtensions;
+import org.jboss.weld.junit5.auto.AddPackages;
+import org.jboss.weld.junit5.auto.EnableAutoWeld;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
 import org.junit.jupiter.api.extension.RegisterExtension;
-import org.sdase.commons.server.weld.testing.WeldAppExtension;
+import org.sdase.commons.server.weld.beans.UsageBean;
 
+@EnableAutoWeld
+@AddPackages(WeldExampleApplication.class)
+@AddExtensions(CdiComponentProvider.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
`PER_CLASS` seems to be necessary because the [Weld JUnit 5 extension](https://github.com/weld/weld-testing/blob/master/junit5/src/main/java/org/jboss/weld/junit5/WeldJunit5Extension.java#L130) will not start the Weld container otherwise. Default Junit 5 lifecycle is `PER_METHOD`.

Reply:
„not start“ is not exactly true. But the Weld container is only available for each test method, which is not suitable for the Dropwizard extension.